gamestonk_terminal/stocks/discovery/disc_controller.py

```diff
@@ -954,6 +954,15 @@ def call_fds(self, other_args: List[str]):
             help="Specify the Equities selection based on the description (not shown in table)",
         )
 
+        parser.add_argument(
+            "-mc",
+            "--marketcap",
+            default=["Large Cap"],
+            nargs="+",
+            dest="marketcap",
+            help="Specify the Equities selection based on Market Cap",
+        )
+
```

Comment:
If I added options, it couldn't handle `" ".join(marketcap).title()` anymore. If there is a workaround please let me know.

Reply:
2 possible workarounds to include choices:

1) Have your market cap be one word (LargeCap,etc..)
2) Input Large/Mid/Small then do `mc = {input} Cap`

Reply:
Apologies if this is the third time commenting, but my comment doesn't show up now.

2 possible workarounds:

1) Market cap to be one work (ie LargeCap)
2) Input just Large/Mid/Small and do something like `mc = f"{input} Cap"`

Reply:
We could remove "cap" from the options and keep just `small` `mid` and `large` since it is implied

Reply:
I removed my comment by mistake. I think we could update from `large cap` to `large` since it is implied no?

Reply:
Don't like the first one because readability. Going to implement the 2nd one, good suggestion. I noticed that the `choices` arguments doesn't make the function as robust, I need to add the option `small` and `Small` unfortunately. By any chance also already encountered that?

Reply:
Found the solution myself, will update now.

Reply:
Nice.  Think you could also set it to str.lower and compare the lower case of each option.

Reply:
True, but in the end in the Finance Database is written 'Large Cap'. You will always require 'large' to be set to 'Large'. Might as well do it right away (otherwise in `financedatabase_view.py` you will still need to add `.title()`)